.gitignore

```diff
@@ -6,3 +6,4 @@
 gdb
 pi3.cfg
 .hafnium_checkout
+secure-flash.img
```

Comment:
Not needed any longer.

Reply:
I reverted the change where I moved the image under out and wrote an answer there, but for easier access I will also copy it here:

"Unfortunately the image will not be accessible by the tests if the file is put under out (checkImgDirAccess fails).
I think the problem is that the FVP runs from the out folder and maybe it makes sure that files are not modified there."

Reply:
Not needed any longer. :-)